regexes/device/mobiles.yml

```diff
@@ -7957,7 +7990,7 @@ Kyocera:
       model: 'KZ $1'
     - regex: 'Kyocera(?:[\-/])?([a-z0-9]+)'
       model: '$1'
-    - regex: '(?:KWC|QC)-([a-z0-9]+)'
+    - regex: '(?:KWC|QC)-?([a-z0-9]+)'
       model: '$1'
```

Comment:
Shouldn't the model names be prefixed with `QC ` or `KWC ` or are the model names numeric only?

Reply:
Yes, they should, I will fix it